### PR TITLE
[skip-ci] [docu] Generate Doxygen docs with Makefile + GitHub Actions 4

### PR DESCRIPTION
**Pull request by couet**
# This Pull request:

replaces https://github.com/root-project/root/pull/16724  which was corrupted after the hackathon changes in `documentation/doxygen`

### CODE REVIEW

File: .github/workflows/root-docs.yml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+        aws configure set default.s3.max_concurrent_requests 200
+        aws configure list
+
+    - name: Sync to S3
+      run: |
+        pwd
+        ls -l ${DOC_LOCATION}
+        aws --endpoint-url https://s3.cern.ch s3 sync --delete ${DOC_LOCATION}/rootdoc_TMP s3://root/doc-test/
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.WS_ACCESS_KEY_ID_WEB }}
```
Comment (dpiparo) [inline .github/workflows/root-docs.yml:239]: Typo, this will not work.

**Comment by dpiparo** [comment]
I committed a change: we'll have to squash before merging *if* at some point we are happy.

**Comment by dpiparo** [comment]
Building ROOT from scratch takes too much time. Could we maybe enable ccache (see master builds) and/or use incremental builds?

**Comment by couet** [comment]
messed up again ..... I will redo tyhe PR

**Comment by couet** [comment]
clean PR here: https://github.com/root-project/root/pull/17298

**Pull request closed without merging**